exercises/03-Polling-SCM/README.md

```diff
@@ -1,8 +1,96 @@
+
+While logged into your Github account, go to https://github.com/settings/tokens.
+
+From here, navigate to **Personal access tokens (classic)** and create a new token. The token should have the scopes below:
+
+- *admin:repo_hook* - for managing hooks (read, write and delete old ones)
+- *repo* - to see private repos
+- *repo:status* - to manipulate commit statuses
+
+Once the token is created, copy it to notepad for use in the next step.
+
+**Step 3: Setup a Jenkins Credential**
+-------------------------------------------
+
+Jenkins has it's own credential store for secrets that need to be used in build pipelines. To add it to Jenkins, go to the Jenkins home page, then click on **Manage Jenkins** > **Security** > **Credentials**
+
+On the credentials page, you can see there are Credential Domains that you can use to scope where credentials can be used, or add new credentials to the Global scope. 
+
+To add a new credential, hover over the label `(Global)` under **Stores scoped to Jenkins**, then click the dropdown box that appears and select **Add credentials**
+
+You can see there's several different kinds of credentials you can store. For this one, you can keep **Username and password** as the kind you'll use for this exercise, and fill out the rest of the fields with the values below:
+
+- Username: Your Github username
+- Password: The Github access token you created in step 2
+- ID: `Github-credentials`
+
+Once that's done, click **Create**
+
+**Step 4: Poll Github from a Freestyle Job**
+-------------------------------------------
+
+Now that the credentials are setup, it's time to use them. Open up your Hello World Freestyle job and open the configure page.
+
+Under the **Source Code Management** section, choose **Git**, then fill out the rest of the fields with the values below:
+
+- Repository URL: The HTTPS URL of your git repository from step 1
+- Credentials: Use the drop-down menu to choose the credentials you created in step 3
+- Branch Specifier: Remove the branch name from this so it will pickup changes from every branch.
+- Under **Build Triggers**, check the box next to **Poll SCM**, then enter `* * * * *` in the corresponding text box. This is a cron expression that will check the repository for a change every minute.
+
+**Step 5: Poll Github from a Pipeline Job**
+-------------------------------------------
+
+Pipeline jobs are a little stricter in terms of what they'll allow configuration-wise. Instead of running the pipeline script from Jenkins as you've been doing, you're going to check it into SCM now.
+
+Go to the pipeline job in your Jenkins console and open the **Configure** page. From here, check the box next to **Poll SCM**, then enter `* * * * *` in the corresponding text box.
+
+Further down in the **Pipeline** section, change `Pipeline Script` to `Pipeline Script from SCM`, then fill out the parameters below.
```

Comment:
after selecting `Pipeline Script from SCM`, you also need to indicate that the scm is git